voyager_archive/views_utils.py

```diff
@@ -8,22 +8,56 @@ def get_auth_record(auth_id: int) -> list:
 
     return get_marc_fields(marc_text)
 
+
 def get_bib_record(bib_id: int) -> list:
     marc_text = get_object_or_404(BibRecord, bib_id=bib_id).bib_record
 
     return get_marc_fields(marc_text)
 
+
 def get_mfhd_record(mfhd_id: int) -> list:
     marc_text = get_object_or_404(MfhdRecord, mfhd_id=mfhd_id).mfhd_record
-   
+
     return get_marc_fields(marc_text)
 
+
 def get_marc_fields(marc_text: str) -> list:
```

Comment:
Missed this before; the return type hint `list` should be changed to `dict`, since this method now returns a dictionary.